Library/ViewModels/SearchViewModel.swift

```diff
@@ -123,6 +126,12 @@ public final class SearchViewModel: SearchViewModelType, SearchViewModelInputs,
       .map { showPopular, popular, searchResults in showPopular ? popular : searchResults }
       .skipRepeats(==)
 
+    let shouldShowNoSearchResults = Signal.combineLatest(isLoading, .merge(clears, projects))
```

Comment:
im a liiiiil bit worried about this combine latest and over-emission problem. wondering if the `clears` are necessary?

maybe we could just add a few more tests to subside my fears. perhaps the situation of typing in a query that brings back projects, and then a new query that doesnt have any results, and making sure this emits properly